src/metadata/metadata.rs

```diff
@@ -126,6 +136,8 @@ impl Metadata {
         self.date = other.date;
         // Editor
         self.editors.extend(other.editors.into_iter());
+        // Editor Term
+        self.editor_term = other.editor_term;
```

Comment:
It seems a bit weird that we unconditionally override the value. That means that the document value will never have any effect. Should this be an `Option<>` instead, and should `"Editor", "Editors"` be the default in case it's `None`?